src/cardLink/CardLink.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+
+import CardLink from '.';
+
+const meta: Meta<typeof CardLink> = {
+  title: 'Navigation/CardLink',
+  component: CardLink,
+  parameters: {
+    docs: {
+      description: {
+        component: "import { CardLink } from '@commerce7/admin-ui'"
+      }
+    }
+  },
+  argTypes: {
+    icon: {
+      description: 'Add an icon to the start of the element',
+      control: 'text',
+      table: {
+        type: { summary: 'string' }
+      }
+    },
+    label: {
+      description: 'The label for the component',
+      control: 'text',
+      table: {
+        type: { summary: 'string' }
+      }
+    },
+    href: {
+      description: 'The url to link to',
+      control: 'text',
+      table: {
+        type: { summary: 'string' }
+      }
+    },
+    children: {
+      description: 'The content of the component',
+      control: 'text',
+      table: {
+        type: { summary: 'ReactNode' }
+      }
+    }
+  }
+};
```

Comment:
@felipesoares6 We are using argTypes here. But why we are not following same patterns in other components?